physics/trajectory_body.hpp

```diff
@@ -265,6 +265,37 @@ Vector<Acceleration, Frame> Trajectory<Frame>::evaluate_intrinsic_acceleration(
 }
 
 template<typename Frame>
+void Trajectory<Frame>::WriteToMessage(
+    not_null<serialization::Trajectory*> const message) const {
+  CHECK(is_root());
+  WriteSubTreeToMessage(message);
+}
+
+template<typename Frame>
+void Trajectory<Frame>::WriteSubTreeToMessage(
+    not_null<serialization::Trajectory*> const message) const {
+  std::unique_ptr<Instant> last_instant = nullptr;
+  serialization::Trajectory::Litter* serialized_litter = nullptr;
```

Comment:
I'd remove `serialized_` everywhere since it's clear that you are doing serialization (there is no "unserialized" litter.)